[2022-07-19 11:43:12,336] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T11:42:00+00:00 [queued]>
[2022-07-19 11:43:12,347] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T11:42:00+00:00 [queued]>
[2022-07-19 11:43:12,348] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 11:43:12,348] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-19 11:43:12,350] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 11:43:12,370] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): respond> on 2022-07-19 11:42:00+00:00
[2022-07-19 11:43:12,376] {standard_task_runner.py:52} INFO - Started process 12078 to run task
[2022-07-19 11:43:12,380] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'respond', 'scheduled__2022-07-19T11:42:00+00:00', '--job-id', '628', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp6jv2nrdw', '--error-file', '/tmp/tmp5z4e7qe5']
[2022-07-19 11:43:12,382] {standard_task_runner.py:80} INFO - Job 628: Subtask respond
[2022-07-19 11:43:12,465] {task_command.py:370} INFO - Running <TaskInstance: simple_dag.respond scheduled__2022-07-19T11:42:00+00:00 [running]> on host 02ffb48252da
[2022-07-19 11:43:12,575] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T11:42:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T11:42:00+00:00
[2022-07-19 11:43:20,524] {logging_mixin.py:115} INFO - (2724, 1)
[2022-07-19 11:43:20,542] {python.py:173} INFO - Done. Returned value was: Greet Responded Again
[2022-07-19 11:43:20,694] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=respond, execution_date=20220719T114200, start_date=20220719T114312, end_date=20220719T114320
[2022-07-19 11:43:20,776] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-19 11:43:20,869] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
